src/app/redux/store.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] [OPTIONAL] You do not need this next line disable action because your project works just fine without it. Kindly remove it.